packages/dds/tree/CHANGELOG.md

```diff
@@ -1,5 +1,79 @@
+
+    We recommend using Node16/Node16 unless absolutely necessary. That will produce transpiled JavaScript that is suitable
+    for use with modern versions of Node.js _and_ Bundlers.
+    [See the TypeScript documentation](https://www.typescriptlang.org/tsconfig#moduleResolution) for more information
+    regarding the module and moduleResolution options.
+
+    **Node10 moduleResolution is not supported; it does not support Fluid Framework's API structuring pattern that is used
+    to distinguish stable APIs from those that are in development.**
+
+### Minor Changes
+
+-   Better events ([#20553](https://github.com/microsoft/FluidFramework/issues/20553)) [35fd3e4b1c](https://github.com/microsoft/FluidFramework/commits/35fd3e4b1cb9bbe42ffdfdc11752b21088abe43d)
+
+    We have updated the Events to make it easier to create granular event listeners for single nodes and better support the
+    undo/redo feature. SharedTree nodes now expose `nodeChanged` and `treeChanged` events that fire in response to changes
+    in the node, and to changes in the subtree rooted at the node, respectively.
+
+    This change was originally made in [#20286](https://github.com/microsoft/FluidFramework/pull/20286) ([ac1e773960](https://github.com/microsoft/FluidFramework/commit/ac1e7739607551abb0dae7fa74dda56aec94b609)).
+
+    [Read more about SharedTree Events at fluidframework.com](https://fluidframework.com/docs/data-structures/tree/#event-handling)
+
+-   Recursive schemas ([#20585](https://github.com/microsoft/FluidFramework/issues/20585)) [4fcfc97a4c](https://github.com/microsoft/FluidFramework/commits/4fcfc97a4c9ba77cc39559a4346bf31e902ef2f8)
+
+    Schemas are even more powerful now with the added support for recursive types, which allows you to define types that
+    reference nodes of the same type in their subtree.
+
+    Users of the beta APIs via `SchemaFactoryRecursive` can now find them on `SchemaFactory`.
+
+    [Read more about Recursive Schema at fluidframework.com](https://fluidframework.com/docs/data-structures/tree/#recursive-schema)
+
+-   Transactions ([#20553](https://github.com/microsoft/FluidFramework/issues/20553)) [35fd3e4b1c](https://github.com/microsoft/FluidFramework/commits/35fd3e4b1cb9bbe42ffdfdc11752b21088abe43d)
+
+    You can group multiple changes such that they are applied atomically, and if they fail, they fail atomically. As a
+    result of grouping changes in a transaction, you also get a single revertible object making it easier to undo and redo.
+
+    [Read more about Transactions at fluidframework.com](https://fluidframework.com/docs/data-structures/tree/#transactions)
+
+-   tree: Empty optional fields on object nodes now are undefined non-enumerable own properties instead of not a property at all. ([#20533](https://github.com/microsoft/FluidFramework/issues/20533)) [7ea644e726](https://github.com/microsoft/FluidFramework/commits/7ea644e7269376d8fb24c6e940069a637b81bf57)
```

Comment:
I think this link would be better:
```suggestion
-   tree: Empty optional fields on object nodes now are undefined non-enumerable own properties instead of not a property at all. ([#20533](https://github.com/microsoft/FluidFramework/issues/20533)) [7ea644e726](https://github.com/microsoft/FluidFramework/commit/7ea644e7269376d8fb24c6e940069a637b81bf57)
```
Yes, thats a 1 character change to the URL.